modules/common/route/route.go

```diff
@@ -99,7 +99,7 @@ func (r *Route) CreateOrPatch(
 	if err != nil {
 		if k8s_errors.IsNotFound(err) {
 			h.GetLogger().Info(fmt.Sprintf("Route %s not found, reconcile in %ds", route.Name, r.timeout))
-			return ctrl.Result{RequeueAfter: time.Duration(r.timeout) * time.Second}, nil
+			return ctrl.Result{RequeueAfter: r.timeout}, nil
```

Comment:
use `%s` tor print the timeout value above

Reply:
done